src/Profiler/NodeVisitor/ProfilerNodeVisitor.php

```diff
@@ -66,7 +67,7 @@ protected function doLeaveNode(Node $node, Environment $env)
 
     private function getVarName(): string
     {
-        return sprintf('__internal_%s', hash('sha256', $this->extensionName));
+        return sprintf('__internal_profiler_%s', $this->varNameSalt++);
```

Comment:
Unless this class is used by multiple extensions, I'm quite sure the `varNameSalt` is not necessary.